iroha/src/asset.rs

```diff
@@ -27,6 +27,18 @@ pub mod isi {
         pub amount: u128,
     }
 
+    impl Instruction for AddAssetQuantity {
+        fn execute(&self, world_state_view: &mut WorldStateView) -> Result<(), String> {
+            world_state_view
+                .world
+                .account(&self.account_id)
+                .unwrap()
+                .assets
+                .insert(self.asset_id.clone(), Asset::new(self.asset_id.clone()));
```

Comment:
Shouldn't AddAssetQty get the current balance and add to it?

Reply:
https://soramitsu.atlassian.net/browse/RH2-45